### ISSUE TITLE
Mob spawners stop spawning mobs correctly at random

### ISSUE THREAD

**Issue by Lambsy** [OmerBenGera/WildStacker#44]
At seemingly random times, some stacked mob spawners stop spawning mobs correctly. When the server is restarted the affected mob spawners function properly again. Not all stacked mob spawners are affected when it happens, only some, and newly placed mob spawners function properly while those affected do not until the server is restarted. This is happening daily on our server.

Video of stacked pig spawners only spawning one pig: https://streamable.com/pu7xw5
Video of stacked iron golem spawners not spawning at all (i was at the location for 30 minutes without a single spawn: https://streamable.com/cg27db

When I use /stacker info on the mob spawner not spawning properly, it displays the correct amount of stacked spawners, so the plugin does know the mob spawner is there and how many there are.

I have been unable to replicate the issue myself, players tell me it happens at times when they unload the chunks and return to the mob spawners. I don't have any other plugins that modify mob spawner behavior. 

Server version: Tuinity 1.16.5
WildStacker version: Build 63
WildStacker config: https://pastebin.com/DyV4654x

**Comment by OmerBenGera** [comment]
Duplicate of #38
